snomed/com.b2international.snowowl.snomed.ql/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,7 @@ Require-Bundle: org.eclipse.equinox.common;bundle-version="3.5.0",
  org.eclipse.emf.ecore,
  org.antlr.runtime,
  org.eclipse.emf.common,
- com.b2international.snowowl.snomed.ecl;visibility:=reexport
+ com.b2international.snomed.ecl;bundle-version="[1.3.0,1.4.0)"
```

Comment:
And here. Note that all three `Require-Bundle` instructions have different version ranges already; I would expect them to have the same value.